app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::UsersController < Admin::BaseController
```

Comment:
@Carmer @bethsebian do you all have any insight into what this means? Via Hound: 
"Use nested module/class definitions instead of compact style."